CONTRIBUTING.md

```diff
@@ -71,6 +72,7 @@ Origin](#developers-certificate-of-origin-11) at the end of this file.*
   reviewers to evaluate the changes.
 - The reviewers have 3 weeks to evaluate the PR and work with the author to
   fix issues and implement improvements.
+- During review there should be no force pushes/rewriting history in the branch.
```

Comment:
@Jacobfaib, I will welcome any suggestions for improving this file so the process is more clear to new developers. Please feel free to edit it directly if you want. Thanks in advance!